### PR TITLE
Version Brazil Portuguese: Felipe Genuino

### PR DESCRIPTION
**Pull request by felipegenuino**
Hi dear, this is my contribution. Version pt-br (portuguese of Brazil).

### CODE REVIEW

**Comment by sergeicodes** [comment]
Hi @felipegenuino, thanks for translating.

You pointed that this translation to Brazil Portuguese and used the URL as `/pt-br`. Are the differences bewteen _Brazil Portuguese_ and _Portugal Portuguese_ that big in written language? Can it be one Portuguese version with `/pt`? Very curious.

In general I don't mind of having both, but for simplicity 1 is better in my opinion. I quickly checked Wikipedia and found that often it has only one Portuguese, so the question is if we can do the same. What do you think?

Thanks!

File: pt-br/index.html [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+          <a target="_blank" href="https://en.wikipedia.org/wiki/Hand">Hands</a
+          >. Hospedado em
+          <a target="_blank" href="https://www.netlify.com/">Netlify</a>. O
+          código fonte está no
+          <a target="_blank" href="https://github.com/sergeicodes/a11ymyths"
+            >GitHub</a
+          >. Acompanhe
+          <a target="_blank" href="https://twitter.com/_sergeikriger"
+            >@_sergeikriger</a
+          >. Translated by <a target="_blank" href="https://www.felipegenuino.com/"
```
Comment (sergeicodes) [inline pt-br/index.html]: @felipegenuino could you translate this.

**Comment by felipegenuino** [comment]
there are some differences between Portuguese from Portugal and Brazilian.
the big difference is in the verbal agreement.  there are some words that
for pt-BR do not make sense.  that's why I reinforce that we must keep the
pt-BR.  Using generic can cause a language barrier, it would give all sides
to understand but some words might sound funny.

Em sex, 10 de dez de 2021 06:35, Sergei Kriger ***@***.***>
escreveu:

**Comment by sergeicodes** [comment]
> that's why I reinforce that we must keep the pt-BR

@felipegenuino fine with me, we can have both versions. Please translate what I mentioned above and we can release.

File: pt-br/index.html (hunk 1 above)
Comment (felipegenuino) [reply pt-br/index.html]: Traduzido por

**Comment by sergeicodes** [comment]
@felipegenuino thanks!

**Pull request merged**